doc/manual/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[project]
+name = "manual"
+version = "0.1.0"
+description = "Add your description here"
```

Comment:
Should we populate this? I see it is not set for the roto project version either.

Reply:
It's not really an issue because we don't publish this Python project, but I added a little description anyway.